app/account/page.jsx

```diff
@@ -0,0 +1,109 @@
+'use client'
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import { useEffect, useState } from 'react'
+import { useUserContext } from '../context/usercontext'
+import { useRouter } from 'next/navigation'
+import Profile from '../components/profile'
+
+function CustomTabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  
```

Comment:
typically you try to avoid having unnecessary new lines in your code so that you can keep things clean and concise. You may want to look at using a linter that adds this rule to prevent unnecessary new lines.